library/systemd/src/lib/yast2/system_service.rb

```diff
@@ -354,7 +352,11 @@ def save(keep_state: false)
       save_start_mode
       perform_action unless keep_state
 
-      errors.none? && reset && refresh!
+      result = errors.none?
+
+      result && reset && refresh
```

Comment:
I prefer more explicit `if result` and not hide it inside boolean expression.

Reply:
I agree